go.mod

```diff
@@ -36,6 +36,7 @@ require (
 	github.com/tetratelabs/getenvoy-package v0.0.0-20190730071641-da31aed4333e
 	github.com/tetratelabs/log v0.0.0-20190710134534-eb04d1e84fb8
 	github.com/tetratelabs/multierror v1.1.0
+	github.com/tetratelabs/proxy-wasm-go-sdk v0.12.0
```

Comment:
one glitch of renaming go.mod -> go.mod_ is that if you do `go mod tidy`, this pops into the root go.mod only because the examples import it. I'll look into it..